piff/basis_interp.py

```diff
@@ -24,6 +24,48 @@
 from .interp import Interp
 from .star import Star
 
+import jax
```

Comment:
Rather than break py3.7 and 3.8, you should try this and gracefully handle cases where jax is unavailable.
```
try:
    import jax
except ImportError:
    can_use_jax = False
else:
    can_use_jax = True
```
Then maybe check below if the user sets `use_jax=True`, but `can_use_jax is False`, give a warning in the log to tell them it's not supported on their system.

This would also mean that jax would be an optional dependency, not a required one, so users can still run Piff without it.  I'm not sure how heavy a dependency jax is, so it might not be that important, but especially as jax is currently not turned on by default, it doesn't seem like we should required it to be installed.

Reply:
Ok sounds good. I will implement it that way.